chcemvediet/apps/inforequests/templates/admin/inforequests/branch/delete_confirmation.html

```diff
@@ -21,7 +21,7 @@
           <ul>{{ delete_constraints|unordered_list }}</ul>
         </div>
       {% endbefore %}
-      {% set_attributes path=".//form//input[@type='submit']" disabled=False %}
+      {% set_attributes path=".//form//input[@type='submit']" disabled=True %}
```

Comment:
@martinmacko47 Pri testovani z predchadzajuceho PR som to nevratil do povodneho stavu. :)

Reply:
@viliambalaz Toto pls fixni v separatnom PR co najskor, kedze je to realny bug a tento PR sa este moze tahat nejaky cas, kym ho odladime.

Reply:
:ok: PR: https://github.com/martinmacko47/chcemvediet/pull/387

Reply:
@viliambalaz Asi ten PR rebasni alebo mergni sem aktualny master. Nech tu mame aktualizovane zmeny z PR mergnutych medzicasom.

Reply:
:ok: